generators/component/templates/functions.php

```diff
@@ -2,27 +2,28 @@
 
 namespace Flynt\Components\<%= nameUpperCamelCase %>;
 
-use Flynt\Api;
+use Flynt\FieldVariables;
 use Flynt\Utils\Options;
 
 add_filter('Flynt/addComponentData?name=<%= nameUpperCamelCase %>', function ($data) {
     return $data;
 });
 
-Api::registerFields('<%= nameUpperCamelCase %>', [
-    'layout' => [
+function getACFLayout()
+{
+    return [
         'name' => '<%= nameLowerCamelCase %>',
         'label' => '<%= namePrettySplit %>',
```

Comment:
sorry, but this would also need the `__()`

Reply:
ah right. Fixed now!